### DISCUSSION TITLE
Can you add the functionality of variant groups

### DISCUSSION THREAD

**Discussion by jipika** [tailwindlabs/tailwindcss#9778] [Ideas]
Something like this
<div class="hover:(bg-gray-400 font-medium) bg-white font-light"/>

**Comment by jipika** [comment]
`<div class="hover:(bg-gray-400 font-medium) bg-white font-light"/>`

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314
